crates/autopilot/src/domain/eth/mod.rs

```diff
@@ -263,7 +263,7 @@ pub struct Event {
 }
 
 /// Any type of on-chain transaction.
-#[derive(Debug)]
+#[derive(Debug, Clone, Default)]
```

Comment:
Why do you need the `Default` here, and in the structs above?

Reply:
needed for settlement unit tests

Reply:
in that case, we could use `#[cfg_attr(test, derive(Default))]`. In production code, those struct shouldn't have a default :thinking:

Reply:
Ah, I see now that I accidentally resolved this comment without fixing. Will address it in another PR.